recipes/pycudadecon/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "pycudadecon" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+
+  url: https://github.com/tlambert03/pycudadecon/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: da21f5a43ca6ebe171d7234fe0e3447bc2956e1d9ebe833da13da0fed00ae3f1
```

```suggestion
  git_url: https://github.com/tlambert03/pycudadecon.git
  git_rev: v{{ version }}
```

lets try this.   sorry for the annoyance... switched to setuptools_scm for versioning, which is what I'm now accustomed to in all my other projects, but those have always pulled from pypi instead of a git tarball.